workflow/rules/multiqc.smk

```diff
@@ -15,6 +15,8 @@ rule FastQC:
         """
         fastqc {input} {params.outdir} -t {threads} 2> {log}
         """
+    # wrapper:
+    #     "v1.25.0/bio/fastqc"
```

Comment:
Error at Runtime:
Error in rule FastQC:
    jobid: 19
    output: results/fastqc/sample97_1_fastqc.html, results/fastqc/sample98_1_fastqc.html, results/fastqc/sample97_2_fastqc.html, results/fastqc/sample98_2_fastqc.html, results/fastqc/sample97_1_fastqc.zip, results/fastqc/sample98_1_fastqc.zip, results/fastqc/sample97_2_fastqc.zip, results/fastqc/sample98_2_fastqc.zip
    log: logs/fastqc/fastq.log (check log file(s) for error message)
    conda-env: /home/tmugoya/PIPELINE/qc_branch/AmpSeeker/.snakemake/conda/94c868735e288e11b98ca93d754edd1d

RuleException:
CalledProcessError in line 19 of /home/tmugoya/PIPELINE/qc_branch/AmpSeeker/workflow/rules/multiqc.smk:
Command 'source /usr/local/anaconda/envs/snakemake/bin/activate '/home/tmugoya/PIPELINE/qc_branch/AmpSeeker/.snakemake/conda/94c868735e288e11b98ca93d754edd1d'; set -euo pipefail;  python /home/tmugoya/PIPELINE/qc_branch/AmpSeeker/.snakemake/scripts/tmpcdhxrfod.wrapper.py' returned non-zero exit status 1.
  File "/home/tmugoya/PIPELINE/qc_branch/AmpSeeker/workflow/rules/multiqc.smk", line 19, in __rule_FastQC
  File "/usr/local/anaconda/envs/snakemake/lib/python3.10/concurrent/futures/thread.py", line 58, in run

Reply:
Fixed by using the shell command instead.

Reply:
@ChabbyTMD Please share the content of logs/fastqc/fastq.log (if not too long) or attach the log file